src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java

```diff
@@ -280,30 +249,20 @@ public static double log10sumLog10(final double[] log10p, final int start) {
     }
 
     public static double log10sumLog10(final double[] log10p, final int start, final int finish) {
-
         if (start >= finish) {
             return Double.NEGATIVE_INFINITY;
         }
-        final int maxElementIndex = MathUtils.maxElementIndex(log10p, start, finish);
+        final int maxElementIndex = maxElementIndex(log10p, start, finish);
         final double maxValue = log10p[maxElementIndex];
         if(maxValue == Double.NEGATIVE_INFINITY) {
             return maxValue;
         }
-        double sum = 1.0;
-        for (int i = start; i < finish; i++) {
-            double curVal = log10p[i];
-            double scaled_val = curVal - maxValue;
-            if (i == maxElementIndex || curVal == Double.NEGATIVE_INFINITY) {
-                continue;
-            }
-            else {
-                sum += Math.pow(10.0, scaled_val);
-            }
-        }
-        if ( Double.isNaN(sum) || sum == Double.POSITIVE_INFINITY ) {
-            throw new IllegalArgumentException("log10p: Values must be non-infinite and non-NAN");
-        }
-        return maxValue + (sum != 1.0 ? Math.log10(sum) : 0.0);
+        final double sum = 1.0 + IntStream.range(start, finish)
+                .filter(i -> i!= maxElementIndex)
```

Comment:
The original author of this code must have thought that `Double.NEGATIVE_INFINITY` is a symbolic object, but it is perfectly well-behaved as mandated by the Java spec.  Here if it is allowed to pass through we get `Math.pow(10.0, Double.NEGATIVE_INIFINITY - maxValue)` which evaluates to `0`, thus not affecting the sum.